templates/layouts/configs/_save-sign-out-button.html

```diff
@@ -24,11 +21,8 @@
                 "data-ga": "click",
                 "data-ga-category": "Navigation",
                 "data-ga-action": "Exit click"
-            },
-            "iconType": "exit",
-            "iconPosition": 'after'
```

Comment:
How is this handled now that it is removed?.

Reply:
These params have now been removed from the header macro in the DS. The save and sign out button is no longer customisable so setting them no longer have any affect. This is the PR that the change was made in: https://github.com/ONSdigital/design-system/pull/2438